src/Evolution/Initialization/Domain.hpp

```diff
@@ -19,18 +20,27 @@
 #include "Domain/Mesh.hpp"
 #include "Domain/MinimumGridSpacing.hpp"
 #include "Domain/Tags.hpp"
+#include "Evolution/Initialization/AddToDataBox.hpp"
 #include "Evolution/Initialization/Helpers.hpp"
+#include "Evolution/Initialization/Tags.hpp"
+#include "Utilities/Requires.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
 
 /// \cond
 template <size_t VolumeDim>
 class ElementIndex;
 namespace Frame {
 struct Inertial;
 }  // namespace Frame
+namespace Parallel {
+template <typename Metavariables>
+class ConstGlobalCache;
+}  // namespace Parallel
 /// \endcond
 
 namespace Initialization {
-
+namespace Actions {
```

Comment:
I suggest to name this `(domain::)Actions::InitializeDomain` and place it in `Domain/Actions/`

Reply:
I'm not sure how I feel about `Domain/Actions`, but it shouldn't be part of this PR.

Reply:
sure 👍